example-plugin/src/test/java/com/yelp/nrtsearch/plugins/example/ExamplePluginTest.java

```diff
@@ -37,41 +40,80 @@
 import org.apache.lucene.analysis.Analyzer;
 import org.apache.lucene.analysis.TokenStream;
 import org.apache.lucene.analysis.tokenattributes.CharTermAttribute;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
 
 public class ExamplePluginTest extends ServerTestCase {
```

Comment:
Is there a reason you are putting the tests in the `example-plugin` instead of the main testing?

Reply:
fixed. Added relevant tests `IngestionPluginUtilsTest`,  `AbstractIngestorTest`